console/src/main/webapp/manager/app/components/area/area.es6

```diff
@@ -121,7 +121,7 @@ class AreaController {
 
       this.updateSelection(selected)
       this.loading = false
-    })
+    }).catch(ex => console.log('Cannot display area. GeoJSON file is malformed or Area config is wrongly configured in the data', ex))
```

Comment:
Asking thought of @tonio on this.

Reply:
I think a broken GeoJSON file will break at line 97.
What about testing this above, & add a specific test for config with an explicit error message ?